tools/topology/topology2/include/common/src_pin_binding.conf

```diff
@@ -0,0 +1,37 @@
+#
+# The source pin binding class definition. All attributes defined herein are
+# namespaced by alsatplg to "Object.Base.src_pin_binding.instance.attribute_name".
+#
+# Usage: Source pin binding objects can be instantiated as
+#
+#	Object.Base.src_pin_binding."0" {
+#		sink_pin_binding_wname	"copier.host.1.0"
```

Comment:
sink or src

Reply:
good catch, copy-paste issue, will update in next push